axelrod/strategies/axelrod_second.py

```diff
@@ -466,3 +467,107 @@ def strategy(self, opponent: Player) -> Action:
             if self.history[-1] == D and opponent_defections_last_8_rounds <= 1:
                 return C
             return D
+
+class Kluepfel(Player):
+    """
+    Strategy submitted to Axelrod's second tournament by Charles Kluepfel
+    (K32R).
+
+    This player keeps track of the the opponent's responses to own behavior:
+
+                      I did:
+                    Def    Coop
+    She does: Coop   C1  |  C3
```

Comment:
Gender neutral please. Go with `player` for your current `I` and `opponent` for your current `she`. Although it's probably also worth changing this to a bullet point list (only because the rest syntax can be troublesome at times). I'd suggest:

```
This player keeps track of the the opponent's responses to own behavior:

- C1 counts: Opponent cooperates as response to player defecting.
- C2 counts: Opponent defects as response to player defecting.
- C3 counts: Opponent cooperates as response to player cooperating.
- C4 counts: Opponent defects as response to player cooperating.
```

Could we however change these variable names (unless there's a good reason) to be more explicit. Perhaps:

```
- `cd_count` counts: Opponent cooperates as response to player defecting.
- `dd_count` counts: Opponent defects as response to player defecting.
- `cc_count` counts: Opponent cooperates as response to player cooperating.
- `dc_count` counts: Opponent defects as response to player cooperating.
```